back/config/tenant_templates/e2etests_template.yml

```diff
@@ -8017,6 +8017,7 @@ models:
       action: budgeting
       permitted_by: groups
       permission_scope_ref: *verifiedPb
+      # Do we need a similar permission here for annotating_document ?
```

Comment:
@alexander-cit WDYT?

If we do need a similar example of such permissions in the `e2etests_template`, do you know how I should add it? (I have completely forgotten).
The [docs](https://www.notion.so/citizenlab/Tenant-templates-c756c55da21e407b97a21519deb96c53?pvs=4#00fe1e4ca14e4e759e3ecc84fa9a4a5d) seem to suggest I edit the template directly and then the [e2etests-template.stg.citizenlab.co](http://e2etests-template.stg.citizenlab.co/) tenant is updated from the template in this repo.

Reply:
Decided we don't need it yet, but only if Brent decides he'd like to add e2e tests of document annotation.